tools/federation/src/main/java/org/eclipse/rdf4j/federated/evaluation/concurrent/ControlledWorkerScheduler.java

```diff
@@ -38,14 +38,14 @@
  */
 public class ControlledWorkerScheduler<T> implements Scheduler<T> {
 
-	protected static final Logger log = LoggerFactory.getLogger(ControlledWorkerScheduler.class);
+	private static final Logger log = LoggerFactory.getLogger(ControlledWorkerScheduler.class);
 
-	protected ExecutorService executor;
+	private ExecutorService executor;
```

Comment:
could executor be final?

Reply:
Sure, I'll add this to @jeenbroekstra's second commit. @hmottestad  Can you please approve once the PR is updated?

Reply:
@hmottestad PR is updated, please have a second look